locales/en.js

```diff
@@ -8,6 +8,8 @@ const en = {
   logout: 'Logout',
   changeUser: 'Change User',
   refreshed: 'Refreshed',
+  notProAlertTitle: '無存取權限!',
+  notProAlertMsg: '是否前往訂閱?',
```

Comment:
This should be in English.

Key to 'premiumFeatureMsg', value to 'Subscribe now'